box/src/Box.styles.ts

```diff
@@ -18,6 +21,8 @@
   background-position: 0 0, 0 0, -0.5rem -0.5rem, 0.5rem 0.5rem;
   position: absolute;
   top: 0px;
+  width: 100%;
+  height: 100%;
```

Comment:
CSS fixes seem unrelated to this PR, normally I expect such fixes to be in individual PRs
here I see a lot of small things like this, this is not good, hard to review, hard to understand from Git history in the future why changes were made